jenkins/scripts/select-compiler.sh

```diff
@@ -239,6 +239,18 @@ elif [ "$SELECT_ARCH" = "ARM64" ]; then
       fi
       echo "Compiler set to GCC" `$CXX -dumpversion`
       ;;
+    *ubuntu2004_sharedlibs* )
```

Comment:
We could possibly make this 
```suggestion
    *ubuntu2004* )
```
but I'm okay with it as-is and update if necessary later on.